src/metabase/util/ssh.clj

```diff
@@ -6,12 +6,20 @@
 
 (def ^:private default-ssh-timeout 30000)
 
+(defn ssh-add-key
+  "Adds SSH private key to Jsch class."
+  [jsch key-file-name]
+  (log/debug "Adding ssh key:" key-file-name)
+  (when-not (string/blank? key-file-name) (doto ^com.jcraft.jsch.JSch jsch (.addIdentity key-file-name)))
```

Comment:
@bcho we do run Eastwood as well as [bikeshed](https://github.com/dakrone/lein-bikeshed) and [docstring checker](https://github.com/camsaul/lein-docstring-checker)